src/main/java/org/cryptomator/linux/tray/AppindicatorTrayMenuController.java

```diff
@@ -0,0 +1,110 @@
+package org.cryptomator.linux.tray;
+
+import org.apache.commons.lang3.SystemUtils;
+import org.cryptomator.integrations.common.CheckAvailability;
+import org.cryptomator.integrations.tray.ActionItem;
+import org.cryptomator.integrations.tray.SeparatorItem;
+import org.cryptomator.integrations.tray.SubMenuItem;
+import org.cryptomator.integrations.tray.TrayMenuController;
+import org.cryptomator.integrations.tray.TrayMenuException;
+import org.cryptomator.integrations.tray.TrayMenuItem;
+import org.purejava.appindicator.MemoryAllocator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.lang.foreign.MemorySegment;
+import java.lang.foreign.SegmentScope;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Paths;
+import java.util.List;
+
+import static org.purejava.appindicator.app_indicator_h.*;
+
+@CheckAvailability
```

Comment:
Also, we should add a Priority annotation, even if the only alternative is currently just a fallback (i.e. lowest possible priority).

But in case of further alternatives are added eventually, it allows us to steer which one is preferred.

Reply:
Ok.